Adding documentation for pure swift package unit testing (#1198)
fastlane/docs: docs/generated/actions/run_tests.md

)
```

```ruby
# run tests on a pure swift package (SPM), no xcodeproj or xcworkspace required
run_tests(

>   package_path: ".",

Reviewer: Perhaps provide a more didactic example here? Like:

```suggestion
  package_path: "/path/to/package/directory" # The path to the directory where your Package.swift is in
```

>   scheme: "MySwiftCode-Package", # Must have -Package

Reviewer: Same here, e.g.:

```suggestion
  scheme: "MySwiftPackage-Package", # Append "-Package" to your Swift Package name
```

  clean: true,
  device: "iPhone 11 Pro", # Required for destination

>   result_bundle: true, # Last two lines may be required for proper xcresults output
>   output_directory: Dir.pwd + "/test_output"

Reviewer: Perhaps this would be better formatted like this?

```suggestion
  # These last two lines may be required for proper .xcresult output:
  result_bundle: true, 
  output_directory: Dir.pwd + "/test_output"
```
